src/js/components/DateInput/DateInput.js

```diff
@@ -244,7 +264,29 @@ Use the icon prop instead.`,
       );
     }
 
-    const calendarButton = (
+    const onClickCopy = () => {
+      global.navigator.clipboard.writeText(textValue);
+      announce(readOnlyCopyValidation, 'assertive');
```

Comment:
Just confirming "assertive" is the correct announce to use? I know that is supposed to be used sparingly

Reply:
Initially I didn't have the 'assertive' set and was not getting the announcement to come through on voice over so I switched to use assertive